common/extensions/api/brave_rewards.json

```diff
@@ -1151,6 +1134,43 @@
           }
         ]
       },
+      {
+        "name": "getAdsStatement",
+        "type": "function",
+        "description": "Get ads statement",
+        "parameters": [
+          {
+            "type": "function",
+            "name": "callback",
+            "parameters": [
+              {
+                "name": "adsStatement",
+                "type": "object",
+                "properties": {
+                  "success": {
+                    "type": "boolean"
+                  },
```

Comment:
It seems a little strange to store the `success` state in the object itself. Usually in a case like this, you can send two arguments back: an integer result and an (optional) object. See how we handle `getPublisherInfo` for example.

Reply:
Agreed, as this is left over from the issue I had on our call, however now that we have resolved that issue I can make this change. Thanks

Reply:
Resolved

Reply:
Fixed